payments-core/src/main/java/com/stripe/android/paymentsheet/paymentdatacollection/BasePaymentDataCollectionFragment.kt

```diff
@@ -0,0 +1,11 @@
+package com.stripe.android.paymentsheet.paymentdatacollection
+
+import androidx.fragment.app.Fragment
+import androidx.fragment.app.viewModels
+import com.stripe.android.paymentsheet.viewmodels.PaymentDataCollectionViewModel
+
+internal abstract class BasePaymentDataCollectionFragment : Fragment() {
+    protected val viewModel: PaymentDataCollectionViewModel by viewModels {
+        PaymentDataCollectionViewModel.Factory { requireActivity().application }
```

Comment:
My understanding is that we prefix with Base classes that are abstract, not to match the fragment/activity name. The ViewModel will not be subclassed, it will be the only one used by the subclasses of `BasePaymentDataCollectionFragment`.